src/test/projects/basic/pom.xml

```diff
@@ -8,8 +8,8 @@
   <packaging>pom</packaging>
 
   <properties>
-    <mavenVersion>3.3.3</mavenVersion>
-    <mavenPluginPluginVersion>3.4</mavenPluginPluginVersion>
+    <mavenVersion>3.6.2</mavenVersion>
```

Comment:
Does this mean maven `3.3.3` is not supported any more?

Reply:
I have reverted the change